src/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/ParallelRangeManager.cs

```diff
@@ -37,6 +41,7 @@ internal struct IndexRange
     /// <summary>
     /// The RangeWorker struct wraps the state needed by a task that services the parallel loop
     /// </summary>
+    [StructLayout(LayoutKind.Auto)]
```

Comment:
what does marking it Auto achieve here? Is it just something you do by default when it's not used for interop?